test/e2e/form-validation.test.js

```diff
@@ -89,6 +89,7 @@ describe('vl-form-validation', async () => {
     it('Als gebruiker zie ik een foutmelding als een rijksregisternummer niet geldig is', async() => {
         const form = await vlFormValidationPage.getFormMetVerplichtRRNVeld();
         await assertThatFormMetInputFieldCorrectValideert(form, '93.05.18-223.61', '93.05.18-223');
+        await assertThatFormMetInputFieldCorrectValideert(form, '88.12.03-001.95', '88.12.03-001.96');
```

Comment:
ene foute ene goeie is genoeg om alle cases te dekken?

Reply:
Ik zal er nog enkele toevoegen voor de zekerheid, nu dek ik inderdaad niet de 2000 validatie.